src/components/CustomNetwork/CustomNetworkDetails.tsx

```diff
@@ -0,0 +1,163 @@
+import React, { useState } from 'react';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import classNames from 'classnames';
+import { Collapse } from 'react-bootstrap';
+import { useSelector } from 'react-redux';
+
+import { CUSTOM_NETWORK_ID } from 'appConstants';
+import { CollapsibleArrows, CopyButton } from 'components';
+import { networks } from 'config';
+import { storage, scrollToElement } from 'helpers';
+import { useGetNetworkChangeLink } from 'hooks';
+import { faTrash, faCheck } from 'icons/regular';
+import { activeNetworkSelector } from 'redux/selectors';
+import { WithClassnameType } from 'types';
+
+const NetworkDetail = ({
+  title,
+  description
+}: {
+  title: string;
+  description: React.ReactNode;
```

Comment:
This could perhaps be declared in an interface, declared above as `NetworkDetailPropsType`?

Reply:
updated in another commit [here](https://github.com/multiversx/mx-explorer-dapp/blob/060810ba5e9198f32924d3b9a4b0005d771a8ca2/src/components/CustomNetwork/CustomNetworkDetails.tsx#L16)